### PR TITLE
saeri training examples

### PR DESCRIPTION
**Pull request by will-moore**
Add examples from:
https://github.com/ome/ome-documentation/pull/2010

NB: These code examples will now be run (checks that they don't throw errors), but this is not strictly the same as being tested.
E.g. we do not check that ```annotation = conn.getObject("Annotation", 1)``` returns anything.
But we have integration tests for that.

### CODE REVIEW

**Comment by will-moore** [comment]
https://merge-ci.openmicroscopy.org/jenkins/job/OMERO-training/246/consoleFull is failing with

```
datasets = conn.getObjects("Dataset", [datasetId, datasetId+1])
01:20:39 TypeError: cannot concatenate 'str' and 'int' objects
```

File: examples/Training/python/Read_Data.py [hunk 1]
```diff
@@ -66,6 +67,32 @@ def print_obj(obj, indent=0):
...
+img_name = img.getName()
+# Can use "Annotation" for all types of annotations by ID
+annotation = conn.getObject("Annotation", 1)
+
+# Find an Object by attribute. E.g. 'name'
+images = conn.getObjects("Image", attributes={"name": img_name})
+
+
+# Get different types of Annotations
+# ===========================
```
Comment (mtbc) [inline examples/Training/python/Read_Data.py]:
```suggestion
# ==================================
```

**Comment by will-moore** [comment]
Training green again today https://py3-ci.openmicroscopy.org/jenkins/job/OMERO-training/75/

**Comment by jburel** [comment]
looks good

**Pull request merged**